libs/blockchain/src/blockchain.config.ts

```diff
@@ -76,6 +77,10 @@ const doRegister = (mode: ChainMode = ChainMode.PROVIDER_SEED_REQUIRED) =>
     }
 
     const configs: JoiUtil.JoiConfig<IBlockchainConfig> = {
+      frequencyTimeoutSecs: {
+        value: process.env.FREQUENCY_TIMEOUT_SECS,
+        joi: Joi.number().positive().default(60),
```

Comment:
nit: Here the default is 60, the environment template files set a value of 10. Do we want these values to be in sync?

Do we want to update the docker-compose.yaml files to get/set FREQUENCY_TIMEOUT_SECS?

Reply:
Good catch. I had set it to 10 in the environment to make testing easier, but that's probably too short. I'll bump it back to 60 in one of my other pending PRs.